### DISCUSSION TITLE
Stream hd version

### DISCUSSION THREAD

**Discussion by AndiLeni** [Motion-Project/motion#1562] [Q&A]
Hello,

I tried to access the full resolution video on the webpage. But the video shown is only the normal, non-hd version.
The config I used is the following:
```
netcam_url rtsp://192.168.178.58:554/stream2
netcam_high_url rtsp://192.168.178.58:554/stream1
```
I would expect the webpage to show the full hd stream instead of the sd when i set `stream_preview_method 0`.
Does this behavior have to be additionally activated?

**Comment by Mr-Dave** [comment]
Just specify the high resolution url as ```netcam_url```

**Reply by AndiLeni** [reply]
Sorry for the late reply. 
If I change the settings as you suggested, the motion detection would work on the hd version?
I tried to avoid this to reduce cpu load.
